models/challenges.js

```diff
@@ -5,6 +5,7 @@
 
 const Firestore = require('@google-cloud/firestore')
 const firestore = require('../utils/firestore')
+const userQuery = require('./users')
```

Comment:
Since we're using only `fetchUser` out of this file, can we just import that? Hint: destructuring